code/datums/components/multitool/circuitboards/circuitboards.dm

```diff
@@ -1,4 +1,4 @@
-/datum/expansion/multitool/circuitboards/CanUseTopic(var/mob/user)
+/datum/component/multitool/circuitboards/CanUseTopic(var/mob/user)
```

Comment:
You really should avoid component subtypes and use either multiple components/elements or add more configuration to the init